gobblin-metrics-libs/gobblin-metrics-base/src/main/java/org/apache/gobblin/metrics/event/TimingEvent.java

```diff
@@ -72,6 +72,8 @@ public static class FlowTimings {
     public static final String FLOW_FAILED = "FlowFailed";
     public static final String FLOW_RUNNING = "FlowRunning";
     public static final String FLOW_CANCELLED = "FlowCancelled";
+    public static final String FLOW_SLA_KILLED = "FlowSLAKilled";
+    public static final String FLOW_START_SLA_KILLED = "FlowStartSLAKilled";
```

Comment:
and overall, I didn't initially see a way to define these semantics w/o adding new `ExecutionStatus` states, but that's what you did, and AFAICT it's a better approach!

Reply:
I was thinking about adding new ExecutionStatuses 😅 but I think there could be more confusion with having too many output states for users if they don't particularly care about the cause. Also it would change the monitoring platforms overall so I wanted to avoid that if possible.